Makefile.rpmbuilder

```diff
@@ -120,7 +120,12 @@ dist-prep:
 dist-build-dep: $(CHROOT_DIR)$(DIST_SRC)/$(PACKAGE)
 
 ifdef INCREMENT_DEVEL_VERSIONS
-dist-package: devel_ver_path = $(ORIG_SRC)/$(OUTPUT_DIR)/$(notdir $(PACKAGE)).devel
+DEVEL_VER_FILE = $(ORIG_SRC)/$(OUTPUT_DIR)/$(notdir $(COMPONENT)).devel
```

Comment:
This change will make all packages of multi-package use a single devel file. Generally I'm fine with this change, but then make sure it is incremented only once (no immediate idea how exactly).

Reply:
I've added several conditions for rebuild. Maybe those could go to Makefile.generic for being available for other builders?